### PR TITLE
Add SpamSecurityInput

### PR DESCRIPTION
**Pull request by gettinToasty**
Now we'll finally have site parity with this input as far as look and feel goes, could be expanded into a TextValueSlider if such a need arises

### CODE REVIEW

File: app/components/widgets/inputs/SpamSecurityInput.vue [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<template>
+  <slider-input :metadata="spamMetadata" :value="localValue" @input="value => updateLocalValue(value)" />
```
Comment (blackxored) [inline app/components/widgets/inputs/SpamSecurityInput.vue]: Can we pass this first-class?

File: app/components/widgets/inputs/SpamSecurityInput.vue.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  localValue = this.optionData[this.value - this.idxMod];
+
+  get optionData() {
+    return [$t('Off'), $t('Low'), $t('Medium'), $t('High'), $t('Very High')];
+  }
+
+  get spamMetadata() {
+    return {
+      data: this.optionData,
+      ...this.metadata,
```
Comment (blackxored) [inline app/components/widgets/inputs/SpamSecurityInput.vue.ts]: Nitpick: Can we put spreads at the top of the object literals?

File: app/services/widgets/settings/alert-box/index.ts [hunk 3]
```diff
@@ -83,20 +83,16 @@ export class AlertBoxService extends WidgetSettingsService<IAlertBoxData> {
...
+        tooltip: $t(
+          'This setting helps control text to speech spam, such as "wwwwwwwwwww." ' +
+            'By default this is turned off, but most users will want to set this to Low or Medium. ' +
+            'The higher the security, the less spam, but also a higher chance of a legitimate message being flagged ' +
+            'as spam and not being read.',
         ),
-        max: 5,
-        min: 1,
-        interval: 1,
+        data: [$t('Off'), $t('Low'), $t('Medium'), $t('High')],
```
Comment (blackxored) [inline app/services/widgets/settings/alert-box/index.ts]: This set of options is used at least twice, wondering if it would make sense to extract.

Comment (gettinToasty) [reply app/services/widgets/settings/alert-box/index.ts]: that's actually a different dataset than the default one.

File: app/components/widgets/inputs/SpamSecurityInput.vue.ts (hunk 2 above)
Comment (gettinToasty) [reply app/components/widgets/inputs/SpamSecurityInput.vue.ts]: in this case: no; we want user-defined data props to overwrite the defaults

File: app/components/widgets/inputs/SpamSecurityInput.vue (hunk 1 above)
Comment (gettinToasty) [reply app/components/widgets/inputs/SpamSecurityInput.vue]: not sure what you mean by that

**Pull request merged**